File: TP1 python/python fede/ejercicio 2/Producto.py
class Producto:
    
    def __init__(self, articulo, dosis, unidadDosis, precio):
        self.articulo = articulo
        self.dosis = dosis
        self.unidadDosis = unidadDosis
        self.precio = precio
        
    def getPrecio(self):
        return self.precio
    
    def imprimir(self):
        print("{}       {}{}       {}".format(self.articulo, self.dosis, self.unidadDosis, self.precio))

File: TP1 java/java fede/Ejercicio 2/Producto.java
public class Producto {
    
    private String  articulo;
    private Integer dosis;
    private String unidadDosis;
    private Double precio;

    public Producto(String articulo, Integer dosis, String unidadDosis, Double precio){
        this.articulo = articulo;
        this.dosis = dosis;
        this.unidadDosis = unidadDosis;
        this.precio = precio;
    }

    public void imprimir(){
        System.out.println(this.articulo + "       " + this.dosis + this.unidadDosis+ "        " + this.precio);
    }

    public Double getPrecio(){
        return this.precio;
    }
}
